.changeset/lemon-numbers-design.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/store-sync": patch
+---
+
+Added boolean array column types for the Postgres decoded indexer.
```

Comment:
```suggestion
Moved boolean array types to use array column types (instead of JSON columns) for the Postgres decoded indexer
```
nit: just to match the changeset I added in https://github.com/latticexyz/mud/pull/2282